src/bench/bench_bitcoin.cpp

```diff
@@ -9,8 +9,7 @@
 #include <validation.h>
 #include <util.h>
 #include <random.h>
-
-#include <boost/lexical_cast.hpp>
+#include <utilstrencodings.h>
```

Comment:
nit: Can run the clang format diff script to sort includes

Reply:
Thanks! Fixed!